03_Modules/SmartCharging/src/module/module.ts

```diff
@@ -68,7 +62,7 @@ export class SmartChargingModule extends AbstractModule {
    * Fields
    */
 
-  protected _requests: CallAction[] = [
+  _requests: CallAction[] = [
```

Comment:
Do we need change for` _responses` as well?

Reply:
removed thank you!